### ISSUE TITLE
[Bug] Fix physics timestep

### ISSUE THREAD

**Issue by anuraghazra** [anuraghazra/Verly.js#30]
Right now Verly.js physics isn't independent of framerate, means if you have a faster or slower PC the physics will behave differently based on framerate. 

Ideally we should take this into account when updating the physics of points/sticks

https://gafferongames.com/post/fix_your_timestep/

**Comment by mreinstein** [comment]
I'm also happy to take on this one

**Comment by anuraghazra** [comment]
Yeah @mreinstein feel free to take any issue you think could improve things.
